src/services/WolfDen.Application/Requests/Commands/LeaveManagement/LeaveTypeCommand/LeaveTypeAddCommandHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿using MediatR;
+using WolfDen.Domain.Entity;
+using WolfDen.Infrastructure.Data;
+
+namespace WolfDen.Application.Requests.Commands.LeaveManagement.LeaveTypeCommand
+{
+    public class LeaveTypeAddCommandHandler : IRequestHandler<LeaveTypeAddCommand, int>
+    {
+        private readonly WolfDenContext _context;
+
+        public LeaveTypeAddCommandHandler(WolfDenContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<int> Handle(LeaveTypeAddCommand request, CancellationToken cancellationToken)
+        {
+            if (request.Id == 1)
```

Comment:
??

Reply:
This was used because , logic for finding HR was not yet told. now it is changed to normal post without checking user